.github/workflows/pr-checklist.yml

```
@@ -8,4 +8,4 @@ jobs:
   pr-checklist:
     runs-on: ubuntu-latest
     steps:
-      - uses: it-at-m/lhm_actions/action-templates/actions/action-pr-checklist@0adbb607339cb32296289c8c319970b9ef2121cb # v1.0.2
\ No newline at end of file
+      - uses: it-at-m/lhm_actions/action-templates/actions/action-pr-checklist@175e2b8e7162b91368e37f1ea5b74bf5ec3ee10a # v1.0.4
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)